.github/workflows/ci-conda-workflow.yml

```
@@ -116,19 +114,18 @@ jobs:
       if: matrix.xspec-version != ''
       env:
         XSPECVER: ${{ matrix.xspec-version }}
+        # Temporarily stop ds9/xpa from installing
+        # on macOS because the tests fail for some
+        # unknown reason
+        OSNAME: {{ matrix.os }}
       run: |
         source ${CONDA}/etc/profile.d/conda.sh
         conda activate build
-        source .github/scripts/setup_ds9.sh
+        if [ "$OSNAME" != "macos-latest" ]; then
```

```suggestion
        if [ "$RUNNER_OS" != "macOS" ]; then
```
I have not tested this in this section of the workflow, but this is used elsewhere in this workflow and would avoid creating an extra var if it works.